crates/rattler_conda_types/src/explicit_environment_spec.rs

```diff
@@ -150,6 +150,27 @@ impl ExplicitEnvironmentSpec {
     pub fn from_path(path: &Path) -> Result<Self, ParseExplicitEnvironmentSpecError> {
         Self::from_reader(File::open(path)?)
     }
+
+    /// Writes an explicit environment spec to file
+    pub fn to_path(&self, path: impl AsRef<Path>) -> Result<(), std::io::Error> {
```

Comment:
It would be nice to have a method that converts this instance to a string. E.g. `to_string()` which is then used by this function. That way people can also write their own serialization logic if they want to.

Reply:
Just to clarify here, do you mean an explicit `to_string` method, or:

```rust
impl fmt::Display for ExplicitEnvironmentSpec {
...
}
```

so that rust automatically implements `ToString`?

Reply:
No I meant a `to_string` method. I dont think it makes sense to implement `Display` because that is expected to be more human-readable.